config/vite_oc10/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "server": "http://host.docker.internal:8080",
+  "theme": "https://host.docker.internal:8081/themes/owncloud/theme.json",
+  "auth": {
+    "clientId": "AWhZZsxb59ouGg97HsdR7GiN8pnzEYvk1cL6aVJgTQH1Gcdxly1gendLVTZ5zpYC",
+    "clientSecret": "VsrTbbeTPJ56e93eKpCdb6Wf5IGHD2meadlsDT1M9EpS3k7Y1ywTYgOhTkKZ0QTL",
+    "url": "http://host.docker.internal:8080/index.php/apps/oauth2/api/v1/token",
+    "authUrl": "http://host.docker.internal:8080/index.php/apps/oauth2/authorize",
+    "logoutUrl": "http://host.docker.internal:8080/index.php/logout"
+  },
+  "apps": [
+    "files",
+    "text-editor",
+    "pdf-viewer",
+    "search"
```

Comment:
`preview` app still doesn't work right?

Reply:
ack, some import/export issue somewhere afaict, just had no time to look into it so far